docs/feature/1692_test_case_based_retries.md

```diff
@@ -0,0 +1,50 @@
+# Test case based retries
+
+## Description
+
+Test case-based retires is a feature request which automatically retries failed tests.
+
+## Things worth mention
+
+- ["One thing that would be worth to keep in mind, since FTL does not support such logic it would require from flank creating new matrix (matrices) for failed tests. It's not something extremely difficult but there is another overhead. FTL will set up new device(s) and that takes some time. I think it's not a blocker but let's don't forget about it."(https://github.com/Flank/flank/issues/778#issuecomment-696027295)
```

Comment:
```suggestion
- ["One thing that would be worth to keep in mind, since FTL does not support such logic it would require from flank creating new matrix (matrices) for failed tests. It's not something extremely difficult but there is another overhead. FTL will set up new device(s) and that will take some time. I think it's not a blocker but let's not forget about it."(https://github.com/Flank/flank/issues/778#issuecomment-696027295)
```

Reply:
this is original quote I would not like to change it